service/accounting/impl/updateCompanyChartOfAccount.go

```diff
@@ -0,0 +1,62 @@
+package impl
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/calvinkmts/expert-pancake/engine/errors"
+	"github.com/calvinkmts/expert-pancake/engine/httpHandler"
+	db "github.com/expert-pancake/service/accounting/db/sqlc"
+	"github.com/expert-pancake/service/accounting/model"
+)
+
+func (a accountingService) UpdateCompanyChartOfAccount(w http.ResponseWriter, r *http.Request) error {
```

Comment:
ini update tapi panggilnya upsert?

Reply:
Ini apa baiknya dipisah query nya?

Reply:
Upsert itu kalau engga ada insert, kalau ada update 

Update ya harus ada dulu baru update.... jadi?

Reply:
Ok brarti di business ada bbrp yang masih gitu juga sih. Soalnya asumsi gk bakal tampil di FE juga harusnya kalau data not exist